docs/hosting/installation/server-setups/docker-compose.md

```diff
@@ -60,51 +60,25 @@ IP address: <IP_OF_YOUR_SERVER>
 Create a `docker-compose.yml` file. Paste the following in the file:
 
 ```yaml
-version: "3"
+version: "3.7"
 
 services:
-  traefik:
-    image: "traefik"
```

Comment:
I would prefer if we kept the Traefik example. No objections to providing a Caddy example in addition.

Reply:
Will change back to Traefik.